packages/core/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@eslint/core",
+  "version": "0.0.1",
+  "description": "Runtime-agnostic core of ESLint",
+  "type": "module",
+  "types": "./src/types.ts",
+  "files": [
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test:jsr": "npx jsr@latest publish --dry-run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eslint/rewrite.git"
+  },
+  "keywords": [
+    "eslint",
+    "core"
+  ],
+  "author": "Nicholas C. Zakas",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/eslint/rewrite/issues"
+  },
+  "homepage": "https://github.com/eslint/rewrite#readme",
```

Comment:
should we probably core's homepage here?

Reply:
We haven't done that for any of the other packages. If we want to do that, we can do so in a separate PR.